tile/lang/compose.cc

```diff
@@ -952,7 +956,11 @@ void FunctionApplication::SetDone() {
     const std::shared_ptr<Value> val = func_->in_bound().at(pi.name);
     if (pi.tag == Input::FIXED) {
       if (val->num_dims() != pi.dims.size()) {
-        throw std::runtime_error("Applying function, tensor with mismatching dimensionality: " + pi.name);
+        throw std::runtime_error(
+            "Applying function, tensor with mismatching dimensionality: " + pi.name +
+            "\n Check the \"Common Issues\" section on our troubleshooting documentation to see if this issue has "
```

Comment:
Are these changes specific to this PR?

Reply:
Similarly, are there cases where we expect a user to hit one of these errors and information in the "Common Issues" section will help them?

Reply:
Our user was running into a `ValueError`, which may be caused by many reasons (of which this is just one).

These changes were actually from my `BatchDot` PR, which is why I'm confused that they appear here.

Reply:
I just copy and paste the up-to-date compose.cc from the website, which has those changes.
Actually, it does not belong to this fix.

Reply:
I think what's needed here is a rebase or merge with master.